### PR TITLE
Add docs for new EDP re:dy component

### PR DESCRIPTION
**Pull request by abmantis**
**Description:**
Docs for a new integration of the EDP re:dy platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#16426

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/edp_redy.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+To set it up, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+edp_redy:
+  username: YOUR_USERNAME
+  password: YOUR_PASSWORD
+```
+
+{% configuration %}
```
Comment (tjorim) [inline source/_components/edp_redy.markdown:31]: Please fix the indentation, remove 2 leading spaces from each line.
https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration

**Review by tjorim — CHANGES_REQUESTED** [review]

File: source/_components/edp_redy.markdown [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+title: "EDP re:dy"
+description: "Instructions on how to integrate the EDP re:dy platform into Home Assistant."
+date: 2018-09-04 16:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: edp_redy.png
+ha_category: Hub
+ha_release: "0.78"
```
Comment (MartinHjelmare) [inline source/_components/edp_redy.markdown]: 0.79

File: source/_components/sensor.edp_redy.markdown [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+title: "EDP re:dy Sensor"
+description: "Instructions on how to integrate the EDP re:dy sensors into Home Assistant."
+date: 2018-09-04 16:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: edp_redy.png
+ha_category: Hub
+ha_release: "0.78"
```
Comment (MartinHjelmare) [inline source/_components/sensor.edp_redy.markdown]: 0.79

File: source/_components/sensor.edp_redy.markdown [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+layout: page
+title: "EDP re:dy Sensor"
+description: "Instructions on how to integrate the EDP re:dy sensors into Home Assistant."
+date: 2018-09-04 16:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: edp_redy.png
+ha_category: Hub
```
Comment (MartinHjelmare) [inline source/_components/sensor.edp_redy.markdown]: Sensor

File: source/_components/switch.edp_redy.markdown [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+layout: page
+title: "EDP re:dy Switch"
+description: "Instructions on how to integrate the EDP re:dy switches into Home Assistant."
+date: 2018-09-04 16:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: edp_redy.png
+ha_category: Hub
```
Comment (MartinHjelmare) [inline source/_components/switch.edp_redy.markdown]: Switch

File: source/_components/switch.edp_redy.markdown [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+ha_release: "0.78"
```
Comment (MartinHjelmare) [inline source/_components/switch.edp_redy.markdown]: 0.79

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**